### PR TITLE
update polyserve. Drop support of Node v4.

### LINKED ISSUE #528: update polyserve to v0.17.0 ...
... & enjoy [all the goodies](https://github.com/Polymer/polyserve/blob/master/CHANGELOG.md#0170-2017-04-13) of the new version

### PR DESCRIPTION
**Pull request by valdrinkoshi**
Fixes #528 
⚠️ breaking change ⚠️  drop support of Node 4 (forced by `polyserve 0.17.0`)
 - [x] CHANGELOG.md has been updated

@aomarks @justinfagnani @rictic FYI

### CODE REVIEW

**Comment by e111077** [comment]
Lovely if this gets merged soon. It's blocking PolymerElements/iron-overlay-behavior#218 from being mergable

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,6 @@ addons:
       - google-chrome-stable
 language: node_js
 node_js:
-  - "4"
   - "6"
```
Comment (rictic) [inline .travis.yml]: please add a list entry for "node"

File: package.json [hunk 2]
```diff
@@ -127,4 +127,4 @@
   "engines": {
     "node": ">= 4.0"
```
Comment (justinfagnani) [inline package.json]: update engines

**Comment by valdrinkoshi** [comment]
Thanks for the review y'all! Feedback incorporated 👌

**Comment by valdrinkoshi** [comment]
ETA for shipping prerelease.8?

**Comment by aomarks** [comment]
Just published 6.0.0-prerelease.8 @next.

**Pull request merged**